test/unit/ajax.js

```diff
@@ -1590,6 +1592,19 @@ module( "ajax", {
 				ok( false, "error" );
 			}
 		} );
+
+		if ( jQuery.support.cors ) {
+			ajaxTest( "gh-1684 - jQuery.ajax() - support cross-domain PATCH in IE10-11", 1, {
+				url: "http://httpbin.org/patch",
```

Comment:
> This will only be a duplicate of the implementation from this patch. I don't want to check if I implemented what I think I did, this is pretty trivial.

This is point of testing as process, point of unit-testing, point of high-coverage tools. It is preferable to test everything.

> We already have such tests - PATCH in IE8 is tested in this way.

I'd say that is not enough.

> IMO a test with a stubbed API would be useless here, it would basically just repeat the code I wrote and wouldn't test anything.

I'm not sure who you come to this conclusion.

> I'd rather not wait for that, it may take a long time and in the meantime we'd have untested changes here.

Their shouldn't be untested, we would need to unit-test it, but your intergration tests in no way better. 

If, for example, Spartan browser wouldn't have a `documentMode` property which is a likely scenario, your tests will not caught anything. Since they are to specific and ua-oriented. 

Now we're introducing another third-party dependency and now requiring to have an internet connection, which we tried really hard to not to do.

Introducing new points of failure to the system is never a good thing.